api/src/utilities/requested-changes-user.ts

```diff
@@ -0,0 +1,16 @@
+import { IdDTO } from 'src/dtos/shared/id.dto';
+import { User } from 'src/dtos/users/user.dto';
+
+/*
+    This maps a user that has requested changes on a listing to a limited IdDTO 
+    This is used by the partner site front end
+  */
+export function requestedChangesUserMapper(user: User): IdDTO {
+  return {
```

Comment:
I wonder if we did a check for `user` and if no user then return `undefined` could speed up our case, since we don't always have a user to map

not sure what your thoughts are, but I think this mapper currently gets called  regardless of if there is a user to transform or not

Reply:
Added a check at the transformation level as to whether or not there is a user to avoid unnecessary calls